recipes/diligent-core/all/conanfile.py

```diff
@@ -184,8 +184,13 @@ def package_info(self):
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Common", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Primitives", "interface"))
         self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "interface"))
+        self.cpp_info.includedirs.append(os.path.join("include", "DiligentCore", "Platforms", "Android", "interface"))
```

Comment:
I suppose we're relying on conan here to prune the non-existing include paths?

Reply:
As far as I know, there is no problem having a non-existing path in PATH variable, while using Windows, Linux or Mac. I can't say anything about mobile OS